etc/nova-agent.service

```diff
@@ -1,12 +1,11 @@
 [Unit]
 Description=Nova Agent for xenstore
-Wants=local-fs.target
 Before=network-pre.target
 After=cloud-init-local.service
```

Comment:
I guess this unit also needs to be after the whatever unit that mounts xenbus or similar...